docs/restic.md

```diff
@@ -32,16 +31,19 @@ cross-volume-type data migrations. Stay tuned as this evolves!
 
 1. Ensure you've [downloaded & extracted the latest release][3].
 
-1. In the Velero directory (i.e. where you extracted the release tarball), run the following to create new custom resource definitions:
+1. Install Velero with restic support
+
+```bash
```

Comment:
Let's highlight that the specific thing they need to do is add the `--use-restic` flag to their `velero install` invocations.  I'd like to add notes around the optional flags that may be relevant (`--prefix`, `--backup-location-config`, `--snapshot-location-config`) -- maybe we can just add links back to the main install pages for each provider, indicating that provider-specific flags will be covered there?

Reply:
Yeah, I'm wondering that, too. Though, each provider has different requirements for configs.

@carlisia What do you think? I know you've expressed concerns about having to look at several documents to get started before. I'd like to avoid duplicating information as much as possible, but also don't want a single, giant page with all the provider's permutations, either.

Reply:
We could just not have setup instructions in `restic.md`, since they're pretty trivial - put the note about the `--use-restic` flag in each of the providers' main setup pages, and here just point to those.

Reply:
Yes, I think these suggestions make sense, in other words, let's try to avoid having duplicate instructions, even if it means having to link people to other pages.

Reply:
Yeah, reviewing this page again, the original idea when restic first came out was to show users how to add it to their existing install. But our main install pages now already mention how to use the flag to set it up; so I'd just simplify this section to say that if you want to use restic, pass that flag during `velero install`, and to see the main install isntructions for more detials.